system/vrops-custom-importer/templates/job.yaml

```
@@ -0,0 +1,50 @@
+---
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: cron-vrops-custom-importer
+  namespace: {{ .Values.namespace }}
+  annotations:
+    "helm.sh/hook": pre-install,pre-upgrade,pre-rollback
+    "helm.sh/hook-delete-policy": before-hook-creation
+    "helm.sh/hook-weight": "3"
+spec:
+  schedule: "{{ .Values.schedule }}"
```

as this seems to be a default in the values, go with this default here and only set the value if something else is specified

```suggestion
  schedule: "{{ default "*/5 * * * *" .Values.schedule }}"
```

leave `.Values.schedule` empty and give it a comment, that it defaults to this.